vaadin-confirm-dialog-flow-parent/vaadin-confirm-dialog-flow/src/main/java/com/vaadin/flow/component/confirmdialog/ConfirmDialog.java

```diff
@@ -375,15 +375,17 @@ public void setRejectButton(String buttonText,
      *            the component to display instead of default Reject button
      */
     public void setRejectButton(Component component) {
-        setRejectButton(component.getElement());
+        SlotUtils.setSlot(this, "reject-button", component.getElement());
```

Comment:
Rather use the `Component` APIs in `SlotUtils.setSlot` also
```suggestion
        SlotUtils.setSlot(this, "reject-button", component);
```

Reply:
Note that there are more than just this once instance. Didn't want to repeat myself so added the comment here only.

Reply:
🤦 Now I wonder if I checked that "allow maintainers to edit" checkbox ;-)

Reply:
That would help. I tried to push earlier but couldn't. But since you work at Vaadin, you should not use your own forks but work with a `flow-components` clone directly. You can request to be added to the `components-contributors` team for write access.